src/components/accordion/CareerAccordion.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  export let open = false
+  import { slide } from 'svelte/transition'
+  import Accordion from './CareerAccordionSection.svelte'
+  const handleClick = () => (open = !open)
```

Comment:
This opens each section independently. Is the intent to make it like the other accordion, where opening one closes the other? Figma is doesn't really tell us. I'm good with doing it this way until told otherwise.